### DISCUSSION TITLE
buddhism/karma/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#13] [General]
# buddhism/karma/

https://shanenull.com/buddhism/karma/

**Comment by shane0** [comment]
Here are some of the sutras about karma:

* The Angulimala Sutta (The Sutta of Angulimala)
* The Anguttara Nikaya (The Book of the Ones)
* The Dhammapada (The Path of Dhamma)
* The Kammavibhanga Sutta (The Sutta on the Analysis of Karma)
* The Majjhima Nikaya (The Book of the Middle Length Discourses)
* The Samyutta Nikaya (The Book of the Connected Discourses)
* The Upajjhatthana Sutta (The Sutta on the Foundations of Mindfulness)

These sutras teach about the law of karma, the different types of karma, and how karma manifests in our lives. They also offer guidance on how to live in a way that creates good karma.

In addition to these sutras, there are many other Buddhist texts that discuss karma, such as the Abhidharma and the Mahayana sutras.

The study of karma is an essential part of Buddhist practice. By understanding karma, we can better understand ourselves and the world around us. We can also learn how to live in a way that leads to happiness and liberation.

https://www.dhammatalks.org/suttas/AN/AN5_57.html
https://www.dhammatalks.org/suttas/MN/
https://www.dhammatalks.org/suttas/SN/
https://www.dhammatalks.org/suttas/MN/MN86.html
https://www.dhammatalks.org/suttas/AN/
https://www.dhammatalks.org/books/KarmaOfMindfulness/Section0012.html

**Comment by shane0** [comment]
http://www.namoamitabha.net/en/publication/liaofan/liaofan_chinkung.pdf

**Comment by shane0** [comment]
https://youtu.be/_W1VFOlkoZo?si=-M5kj2R85hCOWfA2

**Comment by shane0** [comment]
**Karma in the style of Cormac McCarthy:**

Every action has a reaction.

Every cause has an effect.

This is the law of karma.

Karma is the Buddhist teaching that our actions in this life determine our experiences in the next life.

It is not a system of punishment and reward.

It is simply the law of cause and effect.

What we sow, so shall we reap.

If we plant seeds of kindness and compassion, we will reap the fruits of happiness and peace.

If we plant seeds of anger and hatred, we will reap the fruits of suffering and pain.

Karma is not fixed.

We can change our karma through our actions.

If we make the effort to live a life of virtue and compassion, we can create a better future for ourselves.

But we cannot escape the consequences of our actions.

We must face the fruits of our karma, both good and bad.

Karma is a reminder that we are all interconnected.

Our actions affect not only ourselves, but also the people around us.

So let us be mindful of our actions.

Let us strive to plant seeds of kindness and compassion.

And let us create a better future for ourselves and for the world.

**Note:** I have taken some liberties with the traditional Buddhist explanation of karma in order to fit the style of Cormac McCarthy.
